src/Organization/issue.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { IssueProps } from './index'
+
+
+class Issue extends Component<IssueProps> {
+    render() {
+      return (  
+        <div>
+            <span>Author: {this.props.author.login}</span> <br />
```

Comment:
Use de-structuring here to pull out the relevant props:

```
const {title, url, state, author} = this.props
```

Then you will not need to use `this.props`  so much here.